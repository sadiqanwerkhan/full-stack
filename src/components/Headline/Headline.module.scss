@import '../../styles/variables.scss';

.headline {
    font-weight: $font-weight-bold;
    margin-bottom: $spacing-giga;
}