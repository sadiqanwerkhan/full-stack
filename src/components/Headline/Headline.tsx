import { FC, PropsWithChildren } from "react";

import styles from "./Headline.module.scss";

const Headline: FC<PropsWithChildren> = ({ children }) => {
  return <h1 className={`${styles.headline} headlineOne`}>{children}</h1>;
};

export { Headline };
