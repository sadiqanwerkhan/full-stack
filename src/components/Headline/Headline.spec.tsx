import "@testing-library/jest-dom";

import React from "react";

import { Headline } from "./Headline";
import { render } from "@testing-library/react";

describe("Headline", () => {
  it("should render and match styles", () => {
    const { asFragment } = render(<Headline>Headline</Headline>);
    expect(asFragment()).toMatchSnapshot();
  });
});
