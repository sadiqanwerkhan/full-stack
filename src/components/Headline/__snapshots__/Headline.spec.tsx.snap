// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Headline should render and match styles 1`] = `
<DocumentFragment>
  <h1
    class="headline headlineOne"
  >
    Headline
  </h1>
</DocumentFragment>
`;
